Remove italics from japanese

isacoder opened the pull request:
### Description
This PR is to update all the code to check for Japanese and prevent it to be italics

 ### Considerations on Implementation
Current  Stories are based on newest [PR lang implementation](https://github.com/future-standard/scorer-ui-kit/pull/565), but can be updated if rejected.

 ### Reviewing/Testing steps
Storybook -> Form -> Input - Text Input
![Screenshot 2025-03-19 at 12 00 08](https://github.com/user-attachments/assets/82c7661d-f3c5-445d-b294-b87cfdb8eab6)

[Input placeholder video](https://drive.google.com/file/d/1N8lsbCu55CIdsVty_xr7hDo0XSl8pPDe/view?usp=drive_link)

atomworks reviewed (commented):
Everything is looking good here. Are you able to update this based on #565 feedback? Just want to do a sanity check with those things in place before I approve here.

isacoder commented:
I have updated 'us' value to 'en' value as requested and  added the missing text in the BottomDrawer example that updated to show the italics in English and the non-italics in Japanese.

<img width="1427" alt="Screenshot 2025-03-19 at 14 13 51" src="https://github.com/user-attachments/assets/6d68c7e4-ff14-4df7-82ae-274530903c17" />


<img width="1420" alt="Screenshot 2025-03-19 at 14 13 45" src="https://github.com/user-attachments/assets/b1d8b49b-6978-4a19-ade4-e14e6ff0f165" />

atomworks commented on packages/example/src/pages/CustomUserDrawerPage.tsx:
```diff
@@ -45,20 +48,49 @@ const searchPlaceholder = "Search area names, etc.";
     <ExampleText>
-      <p>Custom content can be injected under the user menu like this.</p>
-      <p>The styling for this is left to you to include, keeping it flexible.</p>
+      {attributeLanguage === 'en' ?
```
As we tend to do everything English first, and English becomes our fallback language by default, it would be good to swap the order here, so it's conditional on `ja` else it goes to the English.

isacoder replied:
Thank you for the suggestion, I will update.

atomworks reviewed (approved):
I've added a small comment on something that I was unsure of yesterday but even if it's not updated, this is good. Thank you.
